Tempo/Controllers/PlayerTableViewController.swift

```diff
@@ -95,6 +105,21 @@ class PlayerTableViewController: UITableViewController, UISearchResultsUpdating,
 		removeRevealGesture()
 	}
 	
+	// Filter posted dates into dictionary of key: date, value: date_count
```

Comment:
The reason I had to put it in PlayerTableVC is because PlayerTableVC holds the searching functionality, and since PostHistoryVC has sections the PlayerTableVC needs to be able to call that function to dynamically adjust the contents of the PostHistory child VC. Maybe there's a more efficient way to organize it so that Feed and Liked don't have the variables and function, but that might get messy I'm not certain.